packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -277,16 +284,19 @@ class _TooltipOverlay extends StatelessWidget {
             opacity: animation,
             child: new Opacity(
               opacity: 0.9,
-              child: new Container(
-                decoration: new BoxDecoration(
-                  backgroundColor: Colors.grey[700],
-                  borderRadius: new BorderRadius.circular(2.0)
-                ),
-                height: height,
-                padding: padding,
-                child: new Center(
-                  widthFactor: 1.0,
-                  child: new Text(message, style: Typography.white.body1)
+              child: new Theme(
```

Comment:
none of the descendants here use the theme, and you don't bring in any unknown widgets from the configuration, so you probably don't need to actually embed the theme here.